video_creation/background.py

```diff
@@ -3,29 +3,40 @@
 import re
 from pathlib import Path
 from random import randrange
-from typing import Any, Tuple
+from typing import Any, Tuple,Dict
 
-from moviepy.editor import VideoFileClip
+from moviepy.editor import VideoFileClip,AudioFileClip
 from moviepy.video.io.ffmpeg_tools import ffmpeg_extract_subclip
 from pytube import YouTube
 from pytube.cli import on_progress
 from utils import settings
 from utils.console import print_step, print_substep
+import yt_dlp
```

Comment:
Why use both `yt_dlp` and `pytube`

Reply:
I wrote the code based on the main branch `pytube`. But i was getting error in filter(res=1080p). So i changed the download_video to the same as develop branch `yt_dlp`

Reply:
To my knowledge audio file should not have  `filter(res=1080p` possibly using other filter will not raise any error

Reply:
Yes, you're right. If we remove this filter, the error will not be raised. However, since the `master` branch has this filter in the function, I prefer not to modify this code. My focus in this PR is not on `download_background_video`, but on `audio`.